data-collections-exercises.md

```diff
@@ -3,95 +3,96 @@
 Starting with the following array...
 
 ```rb
-planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+#planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 ```
 
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers.collect {|p| p + " Heart" }
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+# planeteers.delete("Captain Planet") 
+# I hope this what you meant
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+# > hero = planeteers.concat(rangers)
 ```
 
 Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+#  hero.sort!
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+#  hero.shuffle
 ```
 
 #### Bonus
 
 Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+#  hero.sample
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# heroes.select {|str|str.start_with?('b')}
 ```
 
 ### Hashes
 
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+# ninja_turtle = { name: "Michelangelo" ,weapon: "Nunchuks" , radical: true }
 ```
 
 Add a key `age` to `ninja_turtle`. Set it to whatever numerical value you'd like.
 
 ```rb
-# Your answer here
+# ninja_turtle[:age] = 12
 ```
 
 Remove the `radical` key-value pair from `ninja_turtle`.
 
 ```rb
-# Your answer here
+# ninja_turtle.reject { |k,v| k == :radical }
```

Comment:
using this way to solve you won't be able to delete radical directly from ninja_turtle instead you will generate a copy of ninja_turtle hash without the radical key to delete it directly from the hash we use the delete method `ninja_turtle.delete:radical`

Reply:
yeah, you right.
it rejects a copy of ninja_turtle without radical key
I thought it reject radical key form the hash and remove it to different value or hash